### DISCUSSION TITLE
Linux文件共享

### DISCUSSION THREAD

**Discussion by giscus** [ichengzi/ichengzi.github.io#57] [Announcements]
# Linux文件共享

背景：

https://ichengzi.github.io/2022/10/03/linux%E6%96%87%E4%BB%B6%E5%85%B1%E4%BA%AB.html

**Comment by ichengzi** [comment]
fstab 针对的是稳定连接的磁盘， 如果频繁拔插， 不要添加

1. `sudo mount /dev/sdb2 /media/hpan` 设备文件 挂载点
2. `sudo fuser -m /dev/sdb2`  查看设备文件被挂载在哪里
3. `sudo unmont /dev/sdb2`  取消mount， 传入 设备文件或者挂载点都可以

1. `ps 10998` 查看一个进程的detail

**Reply by ichengzi** [reply]
fstab 里添加启动时无法 mount的配置，会导致系统启动失败。 谨记

**Comment by ichengzi** [comment]
mount命令:  https://segmentfault.com/a/1190000023762934

**Comment by ichengzi** [comment]
https://bbs.archlinuxcn.org/viewtopic.php?id=3562

1. `lspci` 查看pci设备
2. `dmesg | grep error` 查看boot异常

**Reply by ichengzi** [reply]
Linux dmesg（英文全称：display message）命令用于显示开机信息。

kernel 会将开机信息存储在 ring buffer 中。您若是开机时来不及查看信息，可利用 dmesg 来查看。开机信息亦保存在 /var/log 目录中，名称为 dmesg 的文件里。

**Comment by ichengzi** [comment]
文件系统损坏，需要进入磁盘只读模式修复。 可以使用安装盘， 演示模式， 修复目标磁盘的文件系统。
https://www.cnblogs.com/conscience-remain/p/15932224.html
